pyccel/codegen/printing/cucode.py

```diff
@@ -100,6 +115,8 @@ def _print_KernelCall(self, expr):
     def _print_CudaSynchronize(self, expr):
         return 'cudaDeviceSynchronize();\n'
 
+    def _print_CudaEmpty(self, expr):
+        return 'cuda_array_create(1,  (int64_t[]){INT64_C(10)}, nd_double, false,allocateMemoryOnHost);\n'
```

Comment:
The memory space and size should not be hard-coded

Reply:
Unused lines should be removed

Reply:
d6ba6ad